src/redux/actions/autorizationAction.ts

```diff
@@ -0,0 +1,11 @@
+import { SET_AUTORIZATION_STATUS_TRUE } from "../constants";
+
+interface SetAutorizationStatusAction {
+  type: string;
+  payload: boolean;
+}
```

Comment:
no need to keep a separate file just for one action.
Also, this SetAutorizationStatusAction interface basically describes all actions
so you may want to rename it and reuse it as generic like:
 
`export enum ActionTypes {
   SET_APP_DATA = "SET_APP_DATA",
   SET_ERROR_ACTION = "SET_ERROR_ACTION",
   SET_AUTORIZATION_STATUS = "SET_AUTORIZATION_STATUS"
}`

`interface Action<Type, Payload> {
     type: Type,
     payload: Payload
}`

then use it like `:Action<ActionTypes.SET_AUTORIZATION_STATUS, boolean>`